bundles/org.openhab.binding.dirigera/src/main/java/org/openhab/binding/dirigera/internal/handler/light/TemperatureLightHandler.java

```diff
@@ -34,15 +40,19 @@
  */
 @NonNullByDefault
 public class TemperatureLightHandler extends DimmableLightHandler {
-    // default values of "standard IKEA lamps" from JSON
-    private int colorTemperatureMax = 2202;
-    private int colorTemperatureMin = 4000;
-    private int range = colorTemperatureMin - colorTemperatureMax;
     private PercentType currentColorTemp = new PercentType();
 
-    public TemperatureLightHandler(Thing thing, Map<String, String> mapping) {
+    protected final DirigeraStateDescriptionProvider stateProvider;
+    // default values of "standard IKEA lamps" from JSON
+    protected int colorTemperatureMin = 4000;
```

Comment:
I still don't understand this. How can colorTemperature**Min** = 4000 Kelvin be a number less than colorTemperature**Max** = 2202 Kelvin ??

(for information "warm" is 2000 K and "daylight" is 6500 K..)

Reply:
Yes, understood. 
Notion needs to be discussed with IKEA as they deliver exactly [these json values ](https://github.com/openhab/openhab-addons/pull/17719/files#diff-c76a1aab29763524d30a873f8867f09d155de63f3716d436fb15fa28458c5213)

![image](https://github.com/user-attachments/assets/607c020b-0999-43f9-8004-464e9fa79940)

Reply:
They probnably see the coldest temperatur as min and warmest as max. This pov is different to openHAB as we look at it from a numeric perspective. 
So it is up to the binding to model the IKEA way to the openHAB pov.

Reply:
Exactly! And it's already working and _thanks to my debug mechanism :)_ you can see in the traces:

High temperature percent - low kelvin value
![image](https://github.com/user-attachments/assets/a20a96fb-1b32-4302-a780-7a93bb83cd56)

Low temperature percent - high kelvin value
![image](https://github.com/user-attachments/assets/7b22c440-c167-46d0-905b-2d8ee120e5d9)

Plus there's [a unit test available](https://github.com/openhab/openhab-addons/blob/cf4156aee5afca2279ba19385a49513697968c27/bundles/org.openhab.binding.dirigera/src/test/java/org/openhab/binding/dirigera/internal/handler/lights/TestTemperatureLight.java#L109) for this behavior. Don't know how to present more prove.

Reply:
Ok. I hear what you say. The OH WebUI has a color temperature slider widget with a visual CT gradation. So I suppose therefore that this gradation will not match what you see on the Ikea lights.

Reply:
@andrewfg are you fine merging this part as is?

Reply:
@andrewfg are you fine merging this as is.

@weymann thanks for your understanding and patience!